spec/index.html

```diff
@@ -1062,7 +1086,7 @@ <h3>The XML Schema Built-in Datatypes</h3>
       <caption>A list of the RDF-compatible XSD types, with short descriptions</caption>
       <tr><th></th><th>Datatype</th><th>Value space (informative)</th></tr>
 
-      <tr><th rowspan="4">Core types</th><td><a data-cite="xmlschema11-2#string"><code>xsd:string</code></a></td><td>Character strings (but not all Unicode character strings)</td></tr>
+      <tr><th rowspan="4">Core types</th><td><a data-cite="xmlschema11-2#string"><code>xsd:string</code></a></td><td>Character strings (see <a>string</a>)</td></tr>
```

Comment:
@afs, @pfps is this correct with the updated definition? I'm not clear on what `xsd:string` literals would not now be considered a string.

Reply:
It's the other way around.  The value space for xsd:string (either version) is a proper subset of Unicode scalar values.  So pointing to the new section on strings gives the impression that xsd:string is larger than it really is.

Reply:
```suggestion
      <tr><th rowspan="4">Core types</th><td><a data-cite="xmlschema11-2#string"><code>xsd:string</code></a></td><td>Character <a>strings</a> matching the <a data-cite="XML11#charsets">Char</a> production from [[XML11]]</td></tr>
```

Reply:
This is better wording but remove the link to `<a>string</a>`

A general (RDF) "string" as used for lexical forms should be independent of XML, and as comprehensive as possible. Anything excluded should have a reason.

Just "Character strings", no link is enough given the link on the previous column.

Or

"Character sequence matching the ..."